docs/index.md

```diff
@@ -4,54 +4,31 @@ page_description: Introduces the Docker Registry
 page_keywords: registry, images, repository
 IGNORES-->
 
-# Docker Registry 2.0
+# Docker Registry
 
-Docker Registry stores and distributes images centrally. It's where you push images to and pull them from; Docker Registry gives team members the ability to share images and deploy them to testing, staging and production environments.
+The Registry is a stateless, highly scalable server side component that stores and lets you distribute Docker images.
 
-Docker provides a hosted registry as part of [Docker Hub](https://hub.docker.com). Docker Hub is a cloud service that securely manages your images. It features organization accounts, automated builds, and much, much more.
+Users looking for a ready-to-go solution are encouraged to head-over to the [Docker Hub](https://hub.docker.com), which does provide a free-to-use, hosted Registry, plus additional features (organization accounts, automated builds, and more).
 
-Docker Registry is the core technology behind the Docker Hub. You can run your own registry instance if you want to host your images privately. Docker Registry features:
-
- - **Pluggable storage drivers**: Images can be stored in Amazon S3, Microsoft Azure or the local filesystem.
- - **Webhook notifications**: When an image is pushed to your registry, webhooks can fire off to launch CI builds, send notifications to IRC, etc.
```

Comment:
This is already going into "understanding the registry" and feels like it belongs to below.